part3/isbn-verifier/spec/isbn_verifier_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "isbn_verifier"
+describe "ISBN-10 Verifier" do
+    it "Returns true if the string is an ISBN-10 format" do
+        expect(isbnVerify("3-598-21508-8")).to eq true
+    end
+    it "Returns false if the string is not an ISBN-10 format" do
```

Comment:
Great start! 
There are a few more edge cases you could check for:

`test_check_digit_is_a_character_other_than_x` -> false

`test_invalid_character_in_isbn` -> false 

`test_valid_isbn_without_separating_dashes` -> true 

ztest_input_is_too_long_but_contains_a_valid_isbn` -> false